tasks/process_agent.py

```diff
@@ -65,6 +65,11 @@ def build(ctx, race=False, go_version=None, incremental_build=False, puppy=False
     ldflags += ' '.join(["-X '{name}={value}'".format(name=main+key, value=value) for key, value in ld_vars.items()])
     build_tags = get_default_build_tags(puppy=puppy)
 
+    ## secrets is not supported on windows because the process agent still runs as
+    ## root.  No matter what `get_default_build_tags()` returns, take secrets out.
+    if sys.platform == 'win32':
+        build_tags.remove("secrets")
```

Comment:
I think this line will actually raise an exception if "secrets" is not in the list of tags, so let's only remove it if it's indeed in the list